src/app/app.component.scss

```diff
@@ -560,3 +548,9 @@ a#cancel1, a#cancel2 {
   display: block !important;
 }
 
+.govuk-link, .govuk-link:visited {
```

Comment:
why do you need to do this. this will keep causing specificity issue. I dont think he should overwrite these css. Let me know what you are trying to achieve